src/pages/developers/tbtc-security-model.md

```diff
@@ -0,0 +1,126 @@
+---
+template: resource
+title: tBTC Security Model
+heading: tBTC Security Model
+description: >-
+  The following is an overview of tBTC’s security model and how it achieves a
+  high level of security for users and signers.
+date: 2020-04-18T18:21:28.438Z
+tags:
+  - security
+---
+tBTC is designed as the safe way for people to use Bitcoin on Ethereum. Different projects take differing approaches to security. The following is an overview of tBTC’s security model and how it achieves a high level of security for users and signers.
+
+## A trust-minimized Bitcoin sidechain
+
+Sidechains were invented as a way to experiment with unlocking Bitcoin’s potential on other chains. The idea is simple - lock BTC on the main Bitcoin blockchain, and unlock it for use on another blockchain, creating a new currency pegged to BTC. In practice, building a 2-way peg without trusted intermediaries is difficult. Blockstream’s Liquid is an example of a Bitcoin sidechain in production that relies on trusted intermediaries.
```

Comment:
This is focused on the security model, rather than describing all of tBTC to someone who is unfamiliar with BTC's security or the idea of sidechains. The beginning of the spec is a good spot to learn this stuff.. open to ideas on how to more generally introduce tBTC in the rest of the docs.